src/web/mjs/connectors/Manga1001Top.mjs

```diff
@@ -0,0 +1,26 @@
+import MadTheme from './templates/MadTheme.mjs';
+
+export default class Manga1001Top extends MadTheme {
+
+    constructor() {
+        super();
+        super.id = 'manga1001top';
+        super.label = 'Manga 1001.top';
+        this.tags = [ 'manga', 'webtoon', 'english' ];
+        this.url = 'https://manga1001.top';
+    }
+
+    async _getChapters(manga) {
+        let mangaid = manga.id.split('/').pop();
+        mangaid = '/'+mangaid;//make sure to take last part of url for the api
+        const uri = new URL('/api/manga'+mangaid+'/chapters?source=detail', this.url);
```

Comment:
`mangaid = '/'+mangaid;` extra instruction adds no value : use `/api/manga/` in the URL